basalt/sdk/promptsdk.py

```
@@ -87,6 +88,63 @@ def get(
             return err, prompt_response, generation
 
         return err, None, None
+        
+    async def async_get(
+        self,
+        slug: str,
+        version: Optional[str] = None,
+        tag: Optional[str] = None,
+        variables: Dict[str, str] = {},
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)